modules/related-posts/related-posts.css

```diff
@@ -53,7 +53,6 @@
 
 .jp-related-posts-i2__post-img-link {
 	order: -1;
-	line-height: 1em;
```

Comment:
How was this change related?

Reply:
Nevermind, saw https://github.com/Automattic/jetpack/pull/13153#issuecomment-516595774 little late. :-)

Was just confused it not mentioned in pr description.